concepticondata/conceptlists/Wilson-2015-173.tsv

```diff
@@ -0,0 +1,174 @@
+Wilson-2015-173-97	97	SNAKE	colubra	culebra	snake	2	Noun	20	3,11	0,04	7,21	730	SNAKE
+Wilson-2015-173-98	98	SNOW	nix	nieve	snow	2	Noun	11	1,62	0,04	21,91	784	SNOW
+Wilson-2015-173-99	99	SOME	paulum	alguno	some	4	Adj.	26	2,57	0,02	1543,82	1241	SOME
+Wilson-2015-173-100	100	STAR	stella	estrella	star	2	Noun	7	0,72	0,04	73,38	1430	STAR
+Wilson-2015-173-101	101	STONE	petra	piedra	stone	1	Noun	21	2,73	0,03	59,85	857	STONE
+Wilson-2015-173-102	102	SUN	sol	sol	sun	0	Noun	8	0,98	0,04	115,74	1343	SUN
+Wilson-2015-173-103	103	TAIL	cauda	cola	tail	2	Noun	28	4,16	0,03	36,18	1220	TAIL
+Wilson-2015-173-104	104	THICK	grossus	grueso	thick	0	Adj.	29	5,41	0,04	36,18	1244	THICK
+Wilson-2015-173-105	105	THIN	tenuis	delgado	thin	4	Adj.	25	2,99	0,03	12,06	2308	THIN
+Wilson-2015-173-106	106	THREE	tres	tres	three	0	Number	1	0,09	0,09	734,41	492	THREE
+Wilson-2015-173-107	107	TO BITE	mordeo	morder	to bite	0	Verb	26	4,67	0,04	17,35	1403	BITE
+Wilson-2015-173-108	108	TO BLOW	flo	soplar	to blow	4	Verb	24	4,31	0,04	8,82	176	BLOW (WITH MOUTH)
+Wilson-2015-173-109	109	TO BREATHE	spiro	respirar	to breathe	4	Verb	18	2,48	0,04	33,68	1407	BREATHE
+Wilson-2015-173-110	110	TO BURN (INTRANSITIVE)	ardeo	quemar	to burn (intransitive)	4	Verb	20	2,94	0,03	33,53	1428	BURNING
+Wilson-2015-173-111	111	TO COME	venio	venir	to come	0	Verb	14	1,9	0,04	1043,53	1446	COME
+Wilson-2015-173-112	112	TO COUNT	numero	contar	to count	4	Verb	27	4,07	0,03	853,97	1420	COUNT
+Wilson-2015-173-113	113	TO CUT	seco	cortar	to cut	4	Verb	23	3,36	0,03	70	1432	CUT
+Wilson-2015-173-114	114	TO DIE	morior	morir	to die	2	Verb	8	0,82	0,04	348,53	1494	DIE
+Wilson-2015-173-115	115	TO DIG	fodio	cavar	to dig	4	Verb	27	3,86	0,03	2,21	1418	DIG
+Wilson-2015-173-116	116	TO DRINK	bibo	beber	to drink	0	Verb	12	1,49	0,03	71,76	1401	DRINK
+Wilson-2015-173-117	117	TO EAT	edo	comer	to eat	4	Verb	15	1,77	0,03	270,29	1336	EAT
+Wilson-2015-173-118	118	TO FALL (DROP)	cado	caer	to fall (drop)	0	Verb	23	2,97	0,03	225,88	1280	FALL
+Wilson-2015-173-119	119	TO FIGHT	pugno	luchar	to fight	4	Verb	29	4,21	0,03	35,29	1423	FIGHT
+Wilson-2015-173-120	120	TO FLOAT	fluito	flotar	to float	5	Verb	27	4,04	0,03	7,06	1574	FLOAT
+Wilson-2015-173-121	121	TO FLOW	fluo	fluir	to flow	0	Verb	27	3,4	0,03	8,24	2003	FLOW
+Wilson-2015-173-122	122	TO FLY	volo	volar	to fly	0	Verb	21	2,83	0,03	53,38	1441	FLY (MOVE THROUGH AIR)
+Wilson-2015-173-123	123	TO GIVE	do	dar	to give	0	Verb	7	0,76	0,04	2657,21	1447	GIVE
+Wilson-2015-173-124	124	TO HEAR	audio	oír	to hear	1	Verb	15	2,35	0,04	471,32	1408	HEAR
+Wilson-2015-173-125	125	TO HIT	verbero	golpear	to hit	4	Verb	40	5,51	0,03	44,41	1433	HIT
+Wilson-2015-173-126	126	TO HUNT (GAME)	venor	cazar	to hunt (game)	4	Verb	23	3,33	0,03	16,03	1435	HUNT
+Wilson-2015-173-127	127	TO KILL	interficio	matar	to kill	4	Verb	28	3,5	0,03	163,97	1417	KILL
+Wilson-2015-173-128	128	TO LAUGH	rideo	reír	to laugh	1	Verb	13	1,61	0,03	106,76	1355	LAUGH
+Wilson-2015-173-129	129	TO LIVE	vivo	vivir	to live	0	Verb	10	1,1	0,04	668,24		
```

Comment:
Could be mapped as `2309 LIVE`.

Reply:
It should be mapped to 1422 BE ALIVE. 2309 LIVE is a duplicate, although I noticed that we mapped two lists to the concept. I'll open a separate topic for this.